Meta/run.py

```diff
@@ -689,7 +689,7 @@ def set_up_network_hardware(config: Configuration):
 
 def set_up_kernel(config: Configuration):
     if config.architecture == Arch.Aarch64:
-        config.kernel_and_initrd_arguments = ["-kernel", "Kernel/Kernel"]
+        config.kernel_and_initrd_arguments = ["-kernel", "Kernel/kernel8.img"]
```

Comment:
This change seems unrelated at this point

Reply:
No, we need to switch to kernel8.img in this commit. I added a reason in the commit message.